src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java

```diff
@@ -0,0 +1,86 @@
+package io.vertx.ext.jdbc.spi;
```

Comment:
For Eclipse Compliance, can you add a copyright header just like the other files?